[0m17:00:24.066777 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11048d8b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1104b2300>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1104b2000>]}


============================== 17:00:24.071793 | 68868df1-fe60-40be-b839-87bf2de04501 ==============================
[0m17:00:24.071793 [info ] [MainThread]: Running with dbt=1.9.1
[0m17:00:24.072121 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': '/Users/nico.laville/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt init my_project', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m17:00:24.080884 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'PosixPath'.. Dictionary: {'dir': PosixPath('/Users/nico.laville/.dbt')}
[0m17:00:24.081131 [info ] [MainThread]: Creating dbt configuration folder at 
[0m17:00:24.082542 [debug] [MainThread]: Starter project path: /Users/nico.laville/Documents/01_DOCS_NICO/05_FORMATION/04_Alternance_datasctientest/04-Cours/DBT/dbt-env/lib/python3.12/site-packages/dbt/include/starter_project
[0m17:00:24.089933 [info ] [MainThread]: 
Your new dbt project "my_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m17:00:24.090181 [info ] [MainThread]: Setting up your profile.
[0m17:02:22.980810 [info ] [MainThread]: Profile my_project written to /Users/nico.laville/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m17:02:23.278458 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 119.26596, "process_in_blocks": "0", "process_kernel_time": 0.195028, "process_mem_max_rss": "109887488", "process_out_blocks": "0", "process_user_time": 0.714176}
[0m17:02:23.281026 [debug] [MainThread]: Command `dbt init` succeeded at 17:02:23.280647 after 119.27 seconds
[0m17:02:23.281968 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1104b2060>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110ba6540>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1109d0350>]}
[0m17:02:23.282697 [debug] [MainThread]: Flushing usage events
[0m17:02:23.722620 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m17:04:03.123547 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10673a420>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10843e210>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10843de80>]}


============================== 17:04:03.126414 | 0cac8324-1a24-492c-b316-81f3aba7aafe ==============================
[0m17:04:03.126414 [info ] [MainThread]: Running with dbt=1.9.1
[0m17:04:03.126758 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/nico.laville/.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt init demo_dbt', 'send_anonymous_usage_stats': 'True'}
[0m17:04:03.136392 [debug] [MainThread]: Starter project path: /Users/nico.laville/Documents/01_DOCS_NICO/05_FORMATION/04_Alternance_datasctientest/04-Cours/DBT/dbt-env/lib/python3.12/site-packages/dbt/include/starter_project
[0m17:04:03.143544 [info ] [MainThread]: 
Your new dbt project "demo_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m17:04:03.143773 [info ] [MainThread]: Setting up your profile.
[0m17:18:09.954034 [info ] [MainThread]: Profile demo_dbt written to /Users/nico.laville/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m17:18:09.960287 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 846.86566, "process_in_blocks": "0", "process_kernel_time": 0.200518, "process_mem_max_rss": "109740032", "process_out_blocks": "0", "process_user_time": 0.738381}
[0m17:18:09.961739 [debug] [MainThread]: Command `dbt init` succeeded at 17:18:09.961436 after 846.87 seconds
[0m17:18:09.962798 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10843e030>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10860c860>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108512db0>]}
[0m17:18:09.963478 [debug] [MainThread]: Flushing usage events
[0m17:18:10.399804 [debug] [MainThread]: An error was encountered while trying to flush usage events
